lib/deck.rb

```diff
@@ -0,0 +1,24 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards #array
+  end
+
+  def count
+    cards.count
+  end
+
+  def current_card(card_num)
+    cards[card_num]
+  end
```

Comment:
We discussed that the `Deck` class doesn't need its own `current_card` method. There is a `current_card` method in the `Round` class that exhibits this functionality.